expression/builtin_other.go

```diff
@@ -263,5 +264,25 @@ type builtinBitCountSig struct {
 
 // See https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#function_bit-count
 func (b *builtinBitCountSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("BIT_COUNT")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return types.Datum{}, errors.Trace(err)
+	}
+	arg := args[0]
+	if arg.IsNull() {
+		return d, nil
+	}
+	sc := new(variable.StatementContext)
+	sc.IgnoreTruncate = true
```

Comment:
mysql> select bit_count("111111111111111111111111111834982309");
+---------------------------------------------------+
| bit_count("111111111111111111111111111834982309") |
+---------------------------------------------------+
|                                                64 |
+---------------------------------------------------+
1 row in set, 1 warning (0.00 sec)